DependencyInjection/Configuration.php

```diff
@@ -27,8 +27,8 @@ public function getConfigTreeBuilder()
     {
         $supportedDrivers = array('orm', 'mongodb');
 
-        $treeBuilder = new TreeBuilder();
-        $rootNode = $treeBuilder->root('pix_sortable_behavior');
+        $treeBuilder = new TreeBuilder('pix_sortable_behavior');
+        $rootNode = $treeBuilder->getRootNode();
```

Comment:
I would add a BC layer for this deprecation notice
```suggestion
        $builder = new TreeBuilder('pix_sortable_behavior');
        if (\method_exists($builder, 'getRootNode')) {
            $rootNode = $builder->getRootNode();
        } else {
            // BC layer for symfony/config 4.1 and older
            $rootNode = $builder->root('pix_sortable_behavior');
        }
```